cGame_logic.cpp

```diff
@@ -694,7 +701,7 @@ bool cGame::setupGame() {
 
 	logger->logHeader("Version information");
 	logger->log(LOG_INFO, COMP_VERSION, "Initializing",
-              fmt::format("Version {}, Compiled at {} , {}", game.m_version, __DATE__, __TIME__));
+              fmt::format("Version {}, Compiled at {} , {} on {}", game.m_version, __DATE__, __TIME__, OS_PLATFORM));
```

Comment:
I think this detail is pretty neat :)

Reply:
Yes but it's not very practical since you have to be independent of the OS as @pbomta  says. And i create a dependency. 

I saw that the SDL2 already had the function to know where we are.
https://wiki.libsdl.org/SDL_GetPlatform

Reply:
I'll leave it up to you how to deal with this. It is not a mandatory feature, but I do think a nice touch. If you think we should not do it, feel free to remove it. (although I think the dependency here is less of a problem compared to the `#ifdef` above?)